runtime/lua/vim/ui.lua

```diff
@@ -71,6 +71,8 @@ end
 ---     - highlight (function)
 ---               Function that will be used for highlighting
 ---               user inputs.
+---     - secret (bool|false)
```

Comment:
```suggestion
---     - secret (boolean)
```

Reply:
`secret (boolean, default false)`, maybe the arguments in `opts` should be validated, like in `vim.fs`?